### ISSUE TITLE
Console shows "Images Optimized", but there are no images in output folder.

### ISSUE THREAD

**Issue by nordskill** [imagemin/imagemin-webp#23]
Hi,
in "images/" I have several .jpg images. I use the code from example and run it in node.js environment:

`const imagemin = require('imagemin');
const imageminWebp = require('imagemin-webp');

imagemin(['images/*.{jpg,png}'], 'images/webp/', {
	use: [
		imageminWebp({quality: 30})
	]
}).then(() => {
	console.log('Images optimized');
});`

After running the script I'm getting 'Images optimized' in console log, nothing happens. The whole process takes less than 1 sec. Are those images are converted, but not saved? Or they're just skipped?

**Comment by BoniWPy** [comment]
yes thats true, me too.

**Comment by mhstoller** [comment]
Yes, the problem does not seem to be with the plugin itself, however. It looks like a possible upstream issue: https://github.com/imagemin/imagemin/issues/327

**Comment by mhstoller** [comment]
In my case it seems that the plugin was unable to support the file type. I'm not sure why no errors were thrown, but I've solved it using different methods and determining the mime type ahead of time....

**Comment by cksho** [comment]
Also example of usage on imagemin-webp is little bit misleading, dunno if imagemin has a new API but below should work:

```javascript
const imagemin = require('imagemin');
const imageminWebp = require('imagemin-webp');

(async () => {
  const files = await imagemin(['./images/*.{jpg}'], {
    destination: 'build/images',
    plugins: [imageminWebp()],
  });

  console.log(files);
})();
```

**Comment by oproulx** [comment]
Had the same problem and @cksho example code solved it.

**Comment by 4k1k0** [comment]
Hello, I already used the `destination` key in the options object, but that only makes a copy of the original `jpg` file I try to convert to `webp`. It doesn't create a new image with `webp` extension.

```
const imagemin = require('imagemin');
const imageminWebp = require('imagemin-webp');

(async () => {
  const img = await imagemin(['images/*.jpg'], {
    destination: 'build/images',
    use: [
      imageminWebp({quality: 100})
    ]
  });

  console.log('Images optimized');
  console.log(img)
})();
```

**Comment by AskingQuestions** [comment]
For those who are having this problem, the `use` keyword is no longer working. `plugins` is the correct option.

```js
const imagemin = require('imagemin');
const imageminWebp = require('imagemin-webp');

(async () => {
  const img = await imagemin(['images/*.jpg'], {
    destination: 'build/images',
    plugins: [
      imageminWebp({quality: 100})
    ]
  });

  console.log('Images optimized');
  console.log(img)
})();
```

**Comment by LambdaZed** [comment]
For Windows users.
You need to replace slahes to unix format
```js
const path = require('path');
const imagemin = require('imagemin');
const imageminWebp = require('imagemin-webp');

(async () => {
  const img = await imagemin([path.resolve(__dirname, 'img/*.{jpg,png}').replace(/\\/g, '/')], {
    destination: path.resolve(__dirname, 'dist/webp').replace(/\\/g, '/'),
    plugins: [imageminWebp({ quality: 70 })],
  });

  console.log('Images optimized');
  console.log(img);
})();
```

**Comment by qwekelly** [comment]
> For Windows users.
> You need to replace slahes to unix format
> 
> ```js
> const path = require('path');
> const imagemin = require('imagemin');
> const imageminWebp = require('imagemin-webp');
> 
> (async () => {
>   const img = await imagemin([path.resolve(__dirname, 'img/*.{jpg,png}').replace(/\\/g, '/')], {
>     destination: path.resolve(__dirname, 'dist/webp').replace(/\\/g, '/'),
>     plugins: [imageminWebp({ quality: 70 })],
>   });
> 
>   console.log('Images optimized');
>   console.log(img);
> })();
> ```

it can‘t work

**Comment by qwekelly** [comment]
it can’t done, and the problem still as usuall @LambdaZed

**Comment by PyGod** [comment]
Hi! Did someone find a solution? It is still a problem...
